src/test/config/bitbucket/values.yaml

```diff
@@ -1,5 +1,7 @@
 # This file contains overrides for the Bitbucket Helm chart's values.yaml file
 
+replicaCount: ${replica.count}
```

Comment:
This is how this value is only applied to Bitbucket, this line is missing in the other products.